include/OpenSpaceToolkit/Astrodynamics/Trajectory/CoordinatesSubset.hpp

```diff
@@ -87,7 +87,7 @@ class CoordinatesSubset
     ///
     /// @return             The resulting coordinates subset value (subset_1 + subset_2)
 
-    VectorXd add(
+    virtual VectorXd add(
```

Comment:
so with this, you can probably remove the implementation in the cpp corresponding file, nah?

Reply:
Yeah this should be added in #156